.github/workflows/ci_common.yml

```diff
@@ -756,3 +756,52 @@ jobs:
         uses: ./
         with:
           subFolder: github-tests/Dockerfile/build-only
+
+  test-platform-with-runcmd:
```

Comment:
Can you also add this job to the `needs` list in the `pr-checks` job?

Reply:
Done!